vgen/src/test/java/io/gapi/vgen/testdata/ruby_library.baseline

```diff
@@ -0,0 +1,471 @@
+            SHELF_PATH_TEMPLATE.instantiate(
+              'shelf' => shelf)
+          end
+
+          # Returns a fully-qualified book resource name string.
+          def self.book_path(shelf, book)
+            BOOK_PATH_TEMPLATE.instantiate(
+              'shelf' => shelf,
+              'book' => book)
+          end
+
+          # @param service_path [String]
+          #   The domain name of the API remote host.
+          # @param port [Integer]
+          #   The port on which to connect to the remote host.
+          # @param channel [Channel]
+          #   A Channel object through which to make calls.
+          # @param ssl_creds [Grpc::ClientCredentials]
+          #   A ClientCredentials for use with an SSL-enabled channel.
+          # @param retrying_override [Hash{String => RetryOptions}]
+          #   A Hash that overrides default retrying settings.
+          #   +retrying_override+ maps method names (e.g., +'list_foo'+) to
+          #   custom RetryOptions objects, or to nil. A value of nil indicates
+          #   that the method in question should not retry.
+          # @param bundling_override [Hash{String => BundleOptions}]
+          #   A Hash that overrides default bundling settings.
+          #   +bundling_override+ maps bundling method names (e.g.,
+          #   +'publish_foo'+) to custom BundleOptions objects, or to nil. It is
+          #   invalid to have a key for a method that is not bundling-enabled. A
+          #   value of nil indicates that the method in question should not
+          #   bundle.
+          # @param timeout [Numeric]
+          #   The default timeout, in seconds, for calls made through this client.
+          # @param app_name [String]
+          #   The codename of the calling service.
+          # @param app_version [String]
+          #   The version of the calling service.
+          def initialize(
+            service_path: SERVICE_ADDRESS,
+            port: DEFAULT_SERVICE_PORT,
+            channel: nil,
+            ssl_creds: nil,
+            scopes: ALL_SCOPES,
+            retrying_override: {},
+            bundling_override: {},
+            timeout: DEFAULT_TIMEOUT,
+            app_name: 'gax',
+            app_version: Google::Gax::VERSION)
+            client_config = Pathname.new(__dir__).join(
+              'library_service_client_config.json')
+            @defaults = client_config.open do |f|
+              Google::Gax.construct_settings(
+                'google.example.library.v1.LibraryService',
+                JSON.parse(f.read),
+                bundling_override,
+                retrying_override,
+                Google::Gax::Grpc::STATUS_CODE_NAMES,
+                timeout,
+                bundle_descriptors: BUNDLE_DESCRIPTORS,
+                page_descriptors: PAGE_DESCRIPTORS)
+            end
+            google_api_client = "#{app_name}/#{app_version} " \
+              "#{CODE_GEN_NAME_VERSION} ruby/#{RUBY_VERSION}".freeze
```

Comment:
This is inconsistent with the Python header, which is semicolon-separated. Is this the format that we're using now?